pages/02.Documentation/02.compute/docs.en.md

```diff
@@ -16,10 +16,10 @@ You can manage your compute instance both via our public [OpenStack API](../../0
 
 ## Flavors
 
-### Standard network storage instance types (M1)
-
-Standard instances generally offer you good performance, availability and storage durability. Disk data will be distributed across multiple servers (SysEleven Distributed Storage).
+### Standard instance types (M1)
 
+Standard instances are equipped with distributed ephemeral (root) storage of a fixed size.
```

Comment:
was war der Grund die Formulierung hier zu verändern, abgesehen von dem Titel?

Reply:
I wanted the descriptions of both flavors in both languages to be comparable and therefor had to change the structure of the old phrase, which we hadn't touched when introducing the new flavor.